tailwind.config.js

```diff
@@ -25,8 +25,8 @@ module.exports = {
         '16': 'repeat(16, minmax(0, 1fr))',
       },
       height: {
```

Comment:
Increased the header height to h-14 to accomodate the h1 size text. h-14 is 3.5rem and so put that here to still have fullscreen be fullscreen